sdcm/nemesis.py

```diff
@@ -4151,9 +4151,7 @@ def disrupt_memory_stress(self):
             self.target_node.install_epel()
             self.target_node.remoter.sudo('yum install -y stress-ng')
         elif self.target_node.distro.is_ubuntu:
-            # Install stress on Ubuntu: https://snapcraft.io/install/stress-ng/ubuntu
-            self.target_node.remoter.sudo('apt -y install snapd')
-            self.target_node.remoter.sudo('snap install stress-ng')
+            self.target_node.remoter.sudo('apt-get -y install stress-ng')
```

Comment:
if package name is identical across distros, let's switch the whole thing to `target_node.install_package()`